src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/UseDefaultDllImportSearchPathsAttributeTests.cs

```diff
@@ -523,7 +523,7 @@ public void TestMethod()
         }
 
         // [DllImport] is set with an absolute path, which will let the [DefaultDllImportSearchPaths] be ignored.
-        [Fact]
+        [Fact(Skip = "user32.dll isn't available on linux")]
```

Comment:
Can we conditionally skip these tests based on OS?

Reply:
I will need to write some custom test attributes for this. Depending on how we feel we can hold off merging until that is done.